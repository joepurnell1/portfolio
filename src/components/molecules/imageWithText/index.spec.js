import React from 'react';
import { ThemeProvider } from 'styled-components';
import renderer from 'react-test-renderer';
import ImageWithText from './index';

const theme = {
  Global: {
    textColour: '#OCE107',
  },
};

describe('ImageWithText component', () => {
  it('should render default as expected', () => {
    const render = renderer.create(
      <ThemeProvider theme={theme}>
        <ImageWithText
          imageSource="http://s3.amazonaws.com/factmag-images/wp-content/uploads/2016/12/131223-run-jewels-christmas-miracle-616x440.jpg"
          imageAlt="LegendHasIt"
          text="I got a unicorn horn for a (Stop)"
        />
      </ThemeProvider>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});
