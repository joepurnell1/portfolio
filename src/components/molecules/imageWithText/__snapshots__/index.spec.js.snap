// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageWithText component should render default as expected 1`] = `
<div
  className="sc-bxivhb kIRBcn"
>
  <img
    alt="LegendHasIt"
    className="sc-bdVaJa PJqeX"
    height="50px"
    src="http://s3.amazonaws.com/factmag-images/wp-content/uploads/2016/12/131223-run-jewels-christmas-miracle-616x440.jpg"
    width="50px"
  />
  <p
    className="sc-ifAKCX jJhMeM sc-htpNat dMmfHA"
  >
    I got a unicorn horn for a (Stop)
  </p>
</div>
`;
